### PR TITLE
#63452 Remove form-link module (and menu item)

### PR DESCRIPTION
**Pull request by Klaas-Ritense**

### CODE REVIEW

File: projects/valtimo/dossier/src/lib/dossier-process-start-modal/dossier-process-start-modal.component.html [hunk 1]
```diff
@@ -34,10 +34,10 @@
       <button
         class="btn btn-secondary btn-space"
         type="button"
-        (click)="gotoFormLinkScreen()"
-        id="form-link-button"
+        (click)="gotoProcessLinkScreen()"
+        id="process-link-button"
```
Comment (teodor-ritense) [inline projects/valtimo/dossier/src/lib/dossier-process-start-modal/dossier-process-start-modal.component.html]: Order according to CODING_GUIDELINES

File: projects/valtimo/dossier/src/lib/dossier-process-start-modal/dossier-process-start-modal.component.ts [hunk 2]
```diff
@@ -99,64 +90,14 @@ export class DossierProcessStartModalComponent implements OnInit {
...
-        });
-        break;
-      default:
-        this.logger.fatal('Unsupported class name');
-    }
+      });
   }
 
-  public gotoFormLinkScreen() {
+  public gotoProcessLinkScreen() {
```
Comment (teodor-ritense) [inline projects/valtimo/dossier/src/lib/dossier-process-start-modal/dossier-process-start-modal.component.ts]:
```suggestion
  public gotoProcessLinkScreen(): void {
```

File: projects/valtimo/dossier/src/lib/dossier-supporting-process-start-modal/dossier-supporting-process-start-modal.component.html [hunk 3]
```diff
@@ -52,10 +52,10 @@
       <button
         class="btn btn-secondary btn-space"
         type="button"
-        (click)="gotoFormLinkScreen()"
-        id="form-link-button"
+        (click)="gotoProcessLinkScreen()"
+        id="process-link-button"
```
Comment (teodor-ritense) [inline projects/valtimo/dossier/src/lib/dossier-supporting-process-start-modal/dossier-supporting-process-start-modal.component.html]: Order according to CODING_GUIDELINES

File: projects/valtimo/dossier/src/lib/dossier-supporting-process-start-modal/dossier-supporting-process-start-modal.component.ts [hunk 4]
```diff
@@ -181,43 +111,25 @@ export class DossierSupportingProcessStartModalComponent {
...
+      });
   }
 
   public formSubmitted() {
     this.modal.hide();
     this.formSubmit.emit();
   }
 
-  public gotoFormLinkScreen() {
+  public gotoProcessLinkScreen() {
```
Comment (teodor-ritense) [inline projects/valtimo/dossier/src/lib/dossier-supporting-process-start-modal/dossier-supporting-process-start-modal.component.ts]:
```suggestion
  public gotoProcessLinkScreen(): void {
```

File: projects/valtimo/task/src/lib/task-detail-modal/task-detail-modal.component.html [hunk 5]
```diff
@@ -57,10 +57,10 @@
       <button
         class="btn btn-secondary btn-space"
         type="button"
-        (click)="gotoFormLinkScreen()"
-        id="form-link-button"
+        (click)="gotoProcessLinkScreen()"
+        id="process-link-button"
```
Comment (teodor-ritense) [inline projects/valtimo/task/src/lib/task-detail-modal/task-detail-modal.component.html]: Order according to CODING_GUIDELINES

**Review by ivo-ritense — APPROVED** [review]

**Pull request merged**